src/main/java/com/zakem/RecipeCatalog/controller/IngredientController.java

```diff
@@ -0,0 +1,41 @@
+package com.zakem.RecipeCatalog.controller;
```

Comment:
In general, it's good practice to separate out the service "model" logic from the Controller logic. Like have an IngredientService and a RecipeService with the methods you need to add / remove.